docs/api/animation/AnimationAction.html

```diff
@@ -11,121 +11,363 @@
+    <div>
+        Sets the [page:.timeScale timeScale] and stops any scheduled warping. This method can be chained.<br /><br />
+        
+        If [page:.paused paused] is false, the effective time scale (an internal property) will be set 
+        to this value, too; otherwise the effective time scale (directly affecting the animation at 
+        this moment) will be set to 0.<br /><br />
+        
+        Note: .*paused* will not be switched to *true* automatically, if .*timeScale* is set to 0 by 
+        this method.
+    </div>    
 
-    <h3>[method:Number getEffectiveTimeScale]()</h3>
-
-    <h3>[method:AnimationAction setDuration]( [page:Number duration] )</h3>
-
-    <h3>[method:AnimationAction syncWith]( [page:AnimationAction action] )</h3>
-
-    <h3>[method:AnimationAction halt]( [page:Number duration] )</h3>
-
-    <h3>[method:AnimationAction warp]( [page:Number startTimeScale], [page:Number endTimeScale], [page:Number duration] )</h3>
-
-    <h3>[method:AnimationAction stopWarping]()</h3>
-
-    <h3>[method:AnimationMixer getMixer]()</h3>
-
-    <h3>[method:AnimationClip getClip]()</h3>
-
-    <h3>[method:Object3D getRoot]()</h3>
-
-
-    <h2>Static Methods</h2>
+    <h3>[method:AnimationAction setEffectiveWeight]( [page:Number weight] )</h3>
+    <div>
+        Sets the [page:.weight weight] and stops any scheduled fading. This method can be chained.<br /><br />
+ 
+        If [page:.enabled enabled] is true, the effective weight (an internal property) will be set to 
+        this value, too; otherwise the effective weight (directly affecting the animation at this 
+        moment) will be set to 0.<br /><br />
+
+        Note: .*enabled* will not be switched to *false* automatically, if .*weight* is set to 0 by 
+        this method.
+    </div>
+    
+    <h3>[method:AnimationAction setLoop]( [page:Number loopMode], [page:Number repetitions] )</h3>
+    <div>
+        Sets the [page:.loop loop mode] and the number of [page:.repetitions repetitions]. This method 
+        can be chained.
+    </div>  
 
+    <h3>[method:AnimationAction startAt]( [page:Number startTimeInSeconds] )</h3>
+    <div>
+        Defines the time for a delayed start (usually passed as [page:AnimationMixer.time] + 
+        deltaTimeInSeconds). This method can be chained.<br /><br />
+        
+        Note: The animation will only start at the given time, if .*startAt* is chained with 
+        [page:.play play], or if the action has already been activated in the mixer (by a previous 
+        call of .*play*, without stopping or resetting it in the meantime).
+    </div>
 
-    <h3>[method:AnimationClip parse]( [page:Object json] )</h3>
+    <h3>[method:AnimationAction stop]()</h3>
+    <div>
+        Tells the mixer to deactivate this action. This method can be chained.<br /><br />
+        
+        The action will be immediately stopped and completely resetted (calling [page:.reset reset] internally).<br /><br />
```

Comment:
`resetted` is not a word in English - the correct form is `reset`. Confusing I know! There are a few cases of this.